doc/compiled.json

```diff
@@ -5727,6 +5727,11 @@
                     "type": "string",
                     "example": "abcd1234abcd1234abcd1234abcd1234"
                   },
+                  "fallback_locale_id": {
+                    "description": "Fallback locale. Can be the name or public id of the locale. Preferred is the public id.",
```

Comment:
I think we can change it to just `id` instead of public id. As an API user there is really only one type of id

Reply:
Maybe let's write something like "Fallback locale for empty translations. Can be a locale name or id"?